apps/dashboard/src/components/primitives/input.tsx

```diff
@@ -59,16 +60,28 @@ const inputFieldVariants = cva(
   }
 );
 
-export type InputFieldProps = { children: React.ReactNode; className?: string } & VariantProps<
+export type InputFieldPureProps = { children: React.ReactNode; className?: string } & VariantProps<
   typeof inputFieldVariants
 >;
 
-const InputField = React.forwardRef<HTMLInputElement, InputFieldProps>(({ children, className, size, state }, ref) => {
-  return (
-    <div ref={ref} className={cn(inputFieldVariants({ size, state }), className)}>
-      {children}
-    </div>
-  );
+const InputFieldPure = React.forwardRef<HTMLInputElement, InputFieldPureProps>(
```

Comment:
It is used for the fields that are not under the form.

Reply:
We need a better name for this. I am not happy with my suggestion. 

```suggestion
const InputFieldNaked = React.forwardRef<HTMLInputElement, InputFieldPureProps>(
```